instrumentation/net/http/otelhttp/internal/body_wrapper.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package internal // import "go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp/internal"
```

Comment:
Can we move these types into packages with names that help remove prefixes? Maybe `otelhttp/internal/request`?

Reply:
Ah, I didn't find a name I was happy with. `request` looks good.